terraform/deployments/cluster-services/aws_auth_configmap.tf

```
@@ -10,7 +10,7 @@
 # opposed to using Helm or kubectl or other tooling designed to work with
 # k8s). This is a rare exception to that rule of thumb.
 
-data "aws_iam_roles" "admin" { name_regex = "(terraform-cloud|\\..*-admin$)" }
+data "aws_iam_roles" "admin" { name_regex = "(terraform-cloud$|\\..*-admin$)" }
```

Does this need the other anchor too? (like this?)
```suggestion
data "aws_iam_roles" "admin" { name_regex = "(^terraform-cloud$|\\..*-admin$)" }
```